### DISCUSSION TITLE
Unable to log into SOC on multiple hosts.

### DISCUSSION THREAD

**Discussion by GankedInAhbazon** [Security-Onion-Solutions/securityonion#4158] [Unsupported Versions]
Hello, I am having issues with my SOC. I am able to log in normally from my stack master laptop but unable to from another laptop connected directly to the stack. When I try to log in to the SOC on the analyst laptop, the screen just flashes and stays on the login screen. I am using Firefox on my master laptop and Google Chrome on the analyst laptop. My only guess at this point is that it is a browser issue.

Additionally, when I try to access the stack via a computer on a network that is being NAT'd into the stack, I get a connection timeout. I have verified that the Centos 7 firewall is not blocking the connection and that the NAT is configured properly. There are also no firewall rules within the stack that are blocking the connection. Using the same external host I can ssh into the manager with no issue using the same IP that I use to access the SOC. I have tried both Google Chrome and Firefox in this instance with no success getting beyond a connection timed out message. I have also ensured to whitelist all IPs that are attempting to connect to the SOC.

I am using version 2.3.40
I installed this from the ISO provided on the Sec Onion website
I chose Centos 7 
Distributed install (Manager, 2 Search, 2 Fwd)
so-status shows all services OK on all nodes
No failures when running salt-call state.highstate

Thanks in advance for any assistance that can be provided. 

Joshua

**Comment by TOoSmOotH** [comment]
Are you using IP or hostname to access it?

**Reply by GankedInAhbazon** [reply]
I was using an IP address. I was able to determine the cause of the problem. I had to edit the /opt/so/saltstack/local/pillar config file and add the public IP to the URL base. This ended up fixing the problem. Even though the NAT was configured properly, the SO manager had issues with the converted IP address being accessed.

**Comment by GankedInAhbazon** [comment]
I was able to determine the cause of this problem and resolved it by adding the NAT'd public IP address after the URL_Base line in the /opt/so/saltstack/local/pillar/global.sls config file.
